engine/src/flutter/impeller/display_list/canvas.cc

```diff
@@ -656,7 +657,7 @@ void Canvas::DrawRoundSuperellipse(const RoundSuperellipse& rse,
                   .AddRoundSuperellipse(rse)
                   .SetBounds(rse.GetBounds())
                   .TakePath();
-  DrawPath(path, paint);
+  DrawPath(flutter::DlPath(path), paint);
```

Comment:
Does this end up converting back to a Skia path (here and above)?

That is probably ok as a first step, but if we're going to end up converting a draw to a path we should do so while recording.

Reply:
We can iterate directly from either SkPath or impeller::Path. The only code that would force a conversion would be `DlPath::GetPath()` or `DlPath::GetSkPath()`. Skia will use GetSkPath() until we stop using Skia, but impeller can switch from GetPath to iterators to avoid any conversions.

Reply:
When I'm done with this work I may be able to delete `DlPath::GetPath() -> impeller::Path`.